frontend/components/DocumentFilters/DocumentFilters.jsx

```diff
@@ -140,9 +166,8 @@ export default class DocumentFilters extends Component {
     const {updateUrlParams} = this.props
 
     let validFilters = filters.filter(entry => {
-      let isValid = Object.keys(entry).filter(key => {
-        return Object.is(entry[key], null)
-      })
+      let isValid = Object.keys(entry)
+        .filter(key => Object.is(entry[key], null))
```

Comment:
@eduardoboucas That depends on whether we'll ever support IE. Edge has full support